src/Compatibility/Microsoft.DotNet.PackageValidation/ReadMe.md

```diff
@@ -0,0 +1,198 @@
+error : API Compatibility errors between lib/netstandard2.0/A.dll (left) and lib/net6.0/A.dll (right):
+error CP0002: Member 'A.B.DoStringManipulation(string)' exists on the left but not on the right.
+```
+
+Finley understands that they shouldn't exclude ```DoStringManipulation(string)``` but instead just provide an additional ```DoStringManipulation(ReadOnlySpan<char>)``` method for .NET 6.0 and changes the code accordingly:
+
+```c#
+#if NET6_0_OR_GREATER
+    public void DoStringManipulation(ReadOnlySpan<char> input)
+    {
+        // use spans to do string operations.
+    }
+#endif
+    public void DoStringManipulation(string input)
+    {
+        // Do some string operations.
+    }
+```
+
+## Validation Against Baseline Package Version
+
+Package Validation helps to validate the latest version of the package with a previous released stable version of the package. This is an opt in feature. The user will need to add the ```EnablePackageBaselineValidation``` to the csproj.
+
+Package validation will detect any breaking changes on any of the shipped target frameworks and will also detect if any target framework support has been dropped.
+
+eg Tom works on the AdventureWorks.Client NuGet package. They want to make sure that they don't accidentally make breaking changes so they configure their project to instruct the package validation tooling to run API compatibility on the previous version of the package.
+
+```csproj
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <PackageVersion>2.0.0</PackageVersion>
+    <EnablePackageBaselineValidation>true</EnablePackageBaselineValidation>
+  </PropertyGroup>
+
+  <Sdk Name="Microsoft.DotNet.PackageValidation" Version="1.0.0-preview4.21111.2" />
+</Project>
+```
+
+A few weeks later, Tom is tasked with adding support for a connection timeout to their library. The Connect method looks like this right now:
+
+```C#
+public static HttpClient Connect(string url)
+{
+    // ...
+}
+```
+
+Since a connection timeout is an advanced configuration setting they reckon they can just add an optional parameter:
+
+```c#
+public static HttpClient Connect(string url, TimeSpan? timeout = default)
+{
+    // ...
+}
+```
+
+However, when they rebuild the package they are getting an error:
+```c#
+error : There are breaking changes between the versions. Please add or modify the apis in the recent version or suppress the intentional breaking changes. 
+error : API compatibility errors in between lib/net6.0/A.dll (left) and lib/net6.0/A.dll (right) for versions 1.0.0 and 2.0.0 respectively:
+error CP0002: Member 'A.B.Connect(string)' exists on the left but not on the right 
+```
+
+Tom realizes that while this is not a source breaking change, it's a binary breaking change. They solve this problem by adding an overload instead:
```

Comment:
Perhaps consider adding links for what a [source breaking change](https://docs.microsoft.com/en-us/dotnet/standard/library-guidance/breaking-changes#source-breaking-change) and [binary breaking change](https://docs.microsoft.com/en-us/dotnet/standard/library-guidance/breaking-changes#binary-breaking-change) mean